### PR TITLE
fixes #65: Lock the 6to5 version to 3.4.1

### LINKED ISSUE #65: Lock the 6to5 version to 3.4.1

### PR DESCRIPTION
**Pull request by ikarienator**
Fixes #65.

### CODE REVIEW

**Comment by michaelficarra** [comment]
Build failed. You have to fix the registration in `package.json`. See shift-ast-js for an example.

**Comment by ikarienator** [comment]
No, our tests were written in ES5. I forgot to update the `build` script and the `common` mode doesn't work.

**Comment by ikarienator** [comment]
Test coverage check failed. Shall we raise the bar or add mode tests?

File: test/literals/literal-regexp-expression.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ suite("Parser", function () {
     testEsprimaEquiv("/[x-z]/i");
     testEsprimaEquiv("/[a-c]/i");
     testEsprimaEquiv("/[P QR]/i");
-    testEsprimaEquiv("/[\\]/]/");
+    testEsprimaEquiv("/[\\]\\/]/");
     testEsprimaEquiv("/foo\\/bar/");
```
Comment (michaelficarra) [inline test/literals/literal-regexp-expression.js]: Rebase on master. This is already fixed.

File: .travis.yml [hunk 2]
```diff
@@ -7,3 +7,6 @@ script:
 node_js:
   - "0.11"
   - "0.10"
+  - "1.0"
+  - "1.1"
+  - iojs
```
Comment (michaelficarra) [inline .travis.yml]: Rebase on master. This is already there. iojs 1.0 and 1.1 will not be tested, just iojs latest.

File: package.json [hunk 3]
```diff
@@ -14,7 +14,8 @@
   ],
   "scripts": {
     "test": "mocha --inline-diffs --check-leaks --ui tdd --reporter dot --recursive test",
-    "build": "6to5 --source-maps-inline --out-dir dist src",
+    "build": "6to5 --modules commonStandard --source-maps-inline --out-dir dist src",
+    "watch": "6to5 --watch --modules commonStandard --source-maps-inline --out-dir dist src",
     "generate-coverage": "istanbul cover _mocha -- --inline-diffs --check-leaks --ui tdd --reporter dot --recursive test",
```
Comment (michaelficarra) [inline package.json]: We can use 6to5 for our tests and during coverage now. Maybe in a separate PR though.

**Comment by michaelficarra** [comment]
I think you mean lower the bar. And yes, do so if necessary.

File: test/literals/literal-regexp-expression.js (hunk 1 above)
Comment (ikarienator) [inline test/literals/literal-regexp-expression.js]: These are what I cherry picked from master.

**Comment by ikarienator** [comment]
Hey! I actually raised the bar.

**Pull request merged**